### PR TITLE
Flow node setup guidance update

### PR DESCRIPTION
**Pull request by franklywatson**

### CODE REVIEW

File: src/content/docs/validator/external-chains/flow.mdx [hunk 1]
```diff
@@ -1,14 +1,43 @@
...
+
+EVM on Flow integrates Ethereum's transaction execution core into Cadence's native transaction runtime and thus Flow's multi
+node consensus. Access Nodes serve the Access API, the native entry point to Flow, accepting Cadence scripts or
+transactions, into which signed transactions from the EVM Gateway are included. The EVM Gateway implements the Ethereum
+JSON-RPC specification for web3.js clients. In the Flow context these two services together represent an Axelar validator.
+Access Nodes, and by extension EVM Gateways, do not vote on the network - they are rpc-only node types.
+
+Operators can choose to run the Access Node and EVM Gateway separately or on the same logical host. Consider that
+Access Nodes are optimized for high throughput script/transaction ingress for Flow clients. Node operators which choose
+to make AN Access APIs public earn increased Flow fees, for which recommended hardware requirements apply. If the validator's
```
Comment (vishalchangrani) [inline src/content/docs/validator/external-chains/flow.mdx:17]:
```suggestion
to make AN Access APIs public, for which recommended hardware requirements apply. If the validator's
```

Comment (vishalchangrani) [reply src/content/docs/validator/external-chains/flow.mdx:17]: ANs don't earn staking rewards.

File: src/content/docs/validator/external-chains/flow.mdx [hunk 2]
```diff
@@ -1,14 +1,43 @@
...
+2. Account creation
+3. Stake your node
+4. Get binaries
+5. Create services
+6. Build EVM Gateway
+7. Start all services
+
+We recommend first time operators to complete the guidance for node setup below in full using testnet.
 
 ## Generate Node Information
```
Comment (vishalchangrani) [inline src/content/docs/validator/external-chains/flow.mdx:42]:
```suggestion
## 1. Generate Node Information
```

File: src/content/docs/validator/external-chains/flow.mdx [hunk 3]
```diff
@@ -1,14 +1,43 @@
...
 - Ubuntu (tested on 18.04 and 20.04)
 - `sudo apt-get install wget liblz4-tool aria2 jq -y`
 - [Official Documentation](https://developers.flow.com/networks/node-ops/access-onchain-data/access-nodes/access-node-setup)
 - [Official Documentation](https://github.com/onflow/flow-evm-gateway/blob/main/README.md)
 - Go
+- Flow CLI (see below)
+
+## Steps
+
+1. Generate node information
```
Comment (vishalchangrani) [inline src/content/docs/validator/external-chains/flow.mdx:32]:
```suggestion
1. Generate access node information
```

**Comment by vishalchangrani** [comment]
I have created a PR with some more changes off of this branch - https://github.com/onflow/axelar-docs/pull/2

**Review by vishalchangrani — APPROVED** [review]

**Review by StephenFluin — APPROVED** [review]

**Pull request merged**